controllers/machineset_controller.go

```diff
@@ -226,6 +227,29 @@ func (r *MachineSetReconciler) reconcile(ctx context.Context, cluster *clusterv1
 		filteredMachines = append(filteredMachines, machine)
 	}
 
+	var errs []error
+	for _, machine := range filteredMachines {
+		if conditions.IsFalse(machine, clusterv1.ConditionOwnerRemediated) {
```

Comment:
Should we test also ConditionHealthCheckSucceeded is false?

Reply:
I think the idea from @vincepri of having two conditions is that remediators would only care about this condition

Reply:
+1, only look at a single condition to keep it simple